LEAlertController/UIAlertView+LEBlocks.m

```diff
@@ -82,4 +92,19 @@ - (void)alertView:(UIAlertView *)alertView clickedButtonAtIndex:(NSInteger)butto
         [originalDelegate alertView:alertView clickedButtonAtIndex:buttonIndex];
 }
 
+- (BOOL)alertViewShouldEnableFirstOtherButton:(UIAlertView *)alertView {
+    
+    BOOL(^shouldEnableFirstOtherButtonBlock)(UIAlertView *alertView) = alertView.shouldEnableFirstOtherButtonBlock;
+    if (shouldEnableFirstOtherButtonBlock) {
+        return shouldEnableFirstOtherButtonBlock(alertView);
+    }
+    
+    id originalDelegate = [self originalDelegate];
+    if (originalDelegate && [originalDelegate respondsToSelector:@selector(alertViewShouldEnableFirstOtherButton:)]) {
```

Comment:
originalDelegate respondsToSelector fails if originalDelegate is nil. So this is redundant.